Fix hh.bib + misc

d97hah opened the pull request:
closes #796

xrotwang commented on languoids/tree/arti1236/toki1239/md.ini:
```diff
@@ -6,6 +6,8 @@ iso639-3 = tok
 countries = 
 	US
+macroareas = 
+	UNKNOWN
```
@d97hah I had fixed (i.e. removed) these before. Maybe you need to fix your side of the machinery that keeps re-inserting?

d97hah commented:
Sorry my bad. Now fixed (in my code and in repos) with PR

Pada tanggal Sab, 21 Mei 2022 pukul 08.28 Robert Forkel <
***@***.***> menulis:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In languoids/tree/arti1236/toki1239/md.ini
> <https://github.com/glottolog/glottolog/pull/850#discussion_r878653475>:
>
> > @@ -6,6 +6,8 @@ iso639-3 = tok
>  hid = tok
>  countries =
>  	US
> +macroareas =
> +	UNKNOWN
>
> @d97hah <https://github.com/d97hah> I had fixed (i.e. removed) these
> before. Maybe you need to fix your side of the machinery that keeps
> re-inserting?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/glottolog/glottolog/pull/850#pullrequestreview-980793478>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA2QHSEERZRQ5XOMXUCNWOLVLB7CNANCNFSM5WQXAFTQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
